### PR TITLE
Update release-notes-azure-cli.md

### PR DESCRIPTION
**Pull request by limingu**

### CODE REVIEW

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -12,6 +12,60 @@ ms.devlang: azurecli
...
+* `dns zone create`: Add auto name server delegation in parent during child zone creation
+
+### Resource
+* `resource link`: Deprecate `--link-id`, `--target-id` and `--filter-string` in favor of
+                   `--link`, `--target`, and `--filter` respectively.
+* `resource link create/update`: Fix issue where these commands would not work.
+* `resource delete`: Fix issue where deleting using a resource ID could crash on error.
+
+### Role
+* [BREAKING CHANGE] 'group create'/'app create'/'sp create-for-rbac': supports idempotent, hence confliction error will not be surfaced
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Unintelligible; sorry, I need to actually know what this change does and what it means for users. __Release notes cannot be merged until this is resolved.__

Comment (limingu) [reply docs-ref-conceptual/release-notes-azure-cli.md]: The breaking change is from PR: https://github.com/Azure/azure-cli/pull/9108.
It fixes 3 issues:
https://github.com/Azure/azure-cli/issues/8750
https://github.com/Azure/azure-cli/issues/8519
https://github.com/Azure/azure-cli/issues/9032

Does it make clear? If not, @yugangw-msft can explain more details as he made the changes.

Comment (sptramer) [reply docs-ref-conceptual/release-notes-azure-cli.md]: Is this truly a breaking change? The issues linked make it look like a bugfix of a feature that was misbehaving. cc @yugangw-msft

Comment (yugangw-msft) [reply docs-ref-conceptual/release-notes-azure-cli.md]: This is a breaking change, for 90% users it is a good break though. I hope the `breaking` wording would reminder them that they no longer need the `try-catch` blocks any more.

Comment (sptramer) [reply docs-ref-conceptual/release-notes-azure-cli.md]: End users do not have to use try-catch blocks. This sounds like it was an engineering-focused change. Please don't include those in the release notes.

Removing.

**Pull request merged**